src/vendor/prism-autolinker.js

```diff
@@ -0,0 +1,76 @@
+;(function() {
```

Comment:
i minified my version of autolink in the file below

Reply:
Did you modify this file yourself or is this an updated version of the autolinker plugin from Prism?